ui/raidboss/data/06-ew/dungeon/aloalo_island.ts

```diff
@@ -67,31 +66,12 @@ const triggerSet: TriggerSet<Data> = {
           ja: 'ドーナツ、玉の下へ',
         },
         third: {
-          en: 'Under Donut Orbs',
-          de: 'Unter Donut Orbs',
-          ja: 'ドーナツ玉の下へ',
-        },
-        text: {
           en: 'Under Donut Far From Axe',
           de: 'Unter Donut und weit weg von der Axt',
           ja: '玉に気を付けて',
         },
       },
     },
-    {
-      id: 'Aloalo Quaqua Arcane Armaments Action',
-      type: 'Ability',
-      netRegex: { id: '8B88', source: 'Quaqua', capture: false },
-      delaySeconds: 2,
-      alertText: (_data, _matches, output) => output.text!(),
-      outputStrings: {
-        text: {
-          en: 'Go to safe NOW!',
```

Comment:
This is very subjective, but in playing through I think the timeline covers the timing this trigger is warning about better than an alert text.